platforms/qt/qt-backend/DatabaseSupport.cpp

```diff
@@ -158,7 +158,7 @@ StackSlot DatabaseConnection::requestDb(RUNNER_ARGS) {
     RUNNER_CheckTag(TString, rawquery);
 
     QString queryString = RUNNER->GetQString(rawquery);
-    QSqlQuery *query = new QSqlQuery(queryString, db);
```

Comment:
This QSqlQuery constructor executes queryString automatically

Reply:
how fun